src/commonTest/kotlin/com/github/michaelbull/result/RunTest.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.michaelbull.result
+
+import kotlinx.coroutines.CoroutineScope
+
+/**
+ * Workaround to use suspending functions in unit tests
```

Comment:
found this solution here: https://github.com/Kotlin/kotlinx.coroutines/issues/885#issuecomment-446586161

Reply:
Might be worth leaving that link in the comment in the code, so we know where it came from and if a fix comes out we can remove it

Reply:
done